backend/controller/sql/schema/20240731230343_create_topic_events_caller.sql

```diff
@@ -0,0 +1,6 @@
+-- migrate:up
+
+ALTER TABLE topic_events
+    ADD COLUMN caller TEXT NOT NULL;
```

Comment:
This will need to have a default (or be nullable), otherwise the migration will fail on existing columns.